### PR TITLE
Added validator to check a field matches another given field

### PR DESCRIPTION
**Pull request by joefitter**
Pass empty array to scripts if no scripts passed. We don't always want the common bundle

Bumped govuk-react-components

### CODE REVIEW

File: pages/common/views/layouts/default.jsx [hunk 1]
```diff
@@ -37,12 +37,14 @@ const Layout = ({
       static: { ...rest, content, urls }
     })
     : {};
-
+  if (scripts.length) {
+    scripts = ['/public/js/common/bundle.js'].concat(scripts);
+  }
```
Comment (lennym) [inline pages/common/views/layouts/default.jsx]: I like this more than I was expecting to.

We just need to be careful that there's no global initialisation that ends up in the bundle.

File: pages/common/assets/sass/style.scss [hunk 2]
```diff
@@ -74,6 +74,10 @@ th {
   display: none;
 }
 
+.grey {
+  color: $grey-1;
+}
+
```
Comment (lennym) [inline pages/common/assets/sass/style.scss]: Does this actually get used anywhere?

Comment (joefitter) [reply pages/common/assets/sass/style.scss]: it _did_

Comment (joefitter) [reply pages/common/assets/sass/style.scss]: In the designs there is a "Step 1 of 2" above the title on the password setup page, but as this is only one step now, I removed it

File: pages/common/views/components/form.jsx [hunk 3]
```diff
@@ -9,6 +9,7 @@ import ConditionalReveal from './conditional-reveal';
 const fields = {
   inputText: props => <Input { ...props } />,
   inputEmail: props => <Input type="email" { ...props } />,
+  inputPassword: props => <Input type="password" { ...props } />,
```
Comment (lennym) [inline pages/common/views/components/form.jsx]: I feel like there's scope here for a `type` prop and then everything is `<Input/>`. Save inventing a new top-level object every time we want a different `type` value.

This works fine too for now though.

**Comment by lennym** [comment]
👍 😄

File: pages/common/views/components/form.jsx (hunk 3 above)
Comment (joefitter) [reply pages/common/views/components/form.jsx]: yeah, agree!

**Pull request merged**